timelock-impl/src/main/java/com/palantir/atlasdb/timelock/adjudicate/FeedbackHandler.java

```diff
@@ -137,30 +144,63 @@ HealthStatus pointFeedbackHealthStatus(ConjureTimeLockClientFeedback healthRepor
                         Constants.START_TRANSACTION_SERVICE_LEVEL_OBJECTIVES)))
                 .filterKeys(Optional::isPresent)
                 .mapKeys(Optional::get)
-                .map((userStats, sloSpec) -> getHealthStatusForService(userStats,
-                        sloSpec.minimumRequestRateForConsideration(),
-                        sloSpec.maximumPermittedP99().toNanos(),
-                        sloSpec.maximumPermittedErrorProportion()))
+                .map((userStats, sloSpec) -> getHealthStatusForMetric(healthReport.getServiceName(),
+                        userStats,
+                        sloSpec))
                 .values()
+                .map(this::logHealthStatusForReport)
                 .max(HealthStatus.getHealthStatusComparator())
                 .orElse(HealthStatus.HEALTHY);
     }
 
-    private HealthStatus getHealthStatusForService(EndpointStatistics endpointStatistics,
-            double rateThreshold,
-            long p99Limit,
-            double errorRateProportion) {
+    private HealthStatus logHealthStatusForReport(
+            PointHealthStatusReport pointHealthStatusReport) {
+        pointHealthStatusReport.message().ifPresent(log::info);
+        return pointHealthStatusReport.status();
+    }
 
-        if (endpointStatistics.getOneMin() < rateThreshold) {
-            return HealthStatus.UNKNOWN;
+    @VisibleForTesting
+    PointHealthStatusReport getHealthStatusForMetric(
+            String serviceName,
+            EndpointStatistics endpointStatistics,
+            ServiceLevelObjectiveSpecification sloSpec) {
+
+        double oneMin = endpointStatistics.getOneMin();
+        if (oneMin < sloSpec.minimumRequestRateForConsideration()) {
+            return PointHealthStatusReport.builder()
+                    .status(HealthStatus.UNKNOWN)
+                    .message(String.format(
+                                    "[%s] | Point health status for %s is UNKNOWN as request rate is low - %.2f",
```

Comment:
We might not want %.2f because of 0.00... values. I'd be okay with just printing all the numbers I think :)

Reply:
👍